discovery/ovhcloud/vps.go

```diff
@@ -37,7 +37,7 @@ const (
 
 // Model struct from API.
 type vpsModel struct {
-	MaximumAdditionalIP int      `json:"maximumAdditionnalIp"`
+	MaximumAdditionalIP int      `json:"maximumAdditionalIp"`
```

Comment:
As this one touches external JSON (un)marshaling, I just want to confirm with @Mama59 / @Arnoways that this is ok? Just in case the external OVH API actually has the field misspelled and we are relying on that or something.

Reply:
The API is actually misspelled, it should stay this way

Reply:
"it should stay this way" meaning it should stay misspelled (as it was), or it should stay corrected (as it currently is in this PR branch)?

Reply:
I think it means the PR shouldn't change this, since it reflects the API.

Reply:
> I think it means the PR shouldn't change this, since it reflects the API.

Correct, the PR should not change this.